pkg/component/observability/logging/fluentoperator/custom_resources.go

```diff
@@ -86,7 +86,7 @@ func (c *customResources) Deploy(ctx context.Context) error {
 		return err
 	}
 
-	return managedresources.CreateForSeed(ctx, c.client, c.namespace, c.getManagedResourceName(), false, serializedResources)
+	return managedresources.CreateForSeedWithLabels(ctx, c.client, c.namespace, c.getManagedResourceName(), false, map[string]string{v1beta1constants.LabelCareConditionType: "ObservabilityComponentsHealthy"}, serializedResources)
```

Comment:
Should we use a common constant for `ObservabilityComponentsHealthy` instead of a string value?

Reply:
Frankly, I was not considering where to put it, but rather have it somewhere instead of always having the string literal with the potential of typos.
As the `LabelCareConditionType` is in `v1beta1constants`, why do we not put it just there?
WDYT?